src/main/java/dk/cngroup/lentils/entity/Role.java

```diff
@@ -18,7 +18,7 @@
     private Long roleId;
 
     @Column(name = "role")
```

Comment:
Dotaz: neměl by se změnit i název sloupce pro sjednocenost?

Reply:
Dobrý dotaz. Ale JPA nevygeneruje správný SQL update, takže bude potřeba manuální oprava v DB. To jsem nechtěl dělat.

JPA vytvoří nový prázdný sloupec name (to je špatný stav) a navíc sloupec role nesmaže.